packages/server/src/authenticator/authByAuthType.ts

```diff
@@ -0,0 +1,19 @@
+import { AuthType, Property, Signature } from '@0auth/message';
+
+import { SecretKey } from '../type';
+import { authPrivacy, authPackage } from '.';
+
+export function authByAuthType(
+  properties: Property[],
+  secret: SecretKey,
+  type: AuthType,
+): Signature {
+  switch (type) {
+    case AuthType.Privacy:
+      return authPrivacy(properties, secret);
+    case AuthType.Package:
+      return authPackage(properties, secret);
+    default:
+      throw new Error('Unreachable Code');
+  }
+}
```

Comment:
The name ʻauthenticator` seems inappropriate.
Wouldn't it be better to just have an ʻissuer` folder instead of an ʻauthenticator`?